weather.go

```diff
@@ -53,18 +51,17 @@ func CliOutput(token, location string) (output string) {
 	}
 	json.Unmarshal(weatherString, &r)
 
-	output = string(fmt.Sprintf("weather: %s\ncelcius: %v\n", r.OneWord, r.Celcius))
+	output = string(fmt.Sprintf("city: %s\nweather: %s\ncelcius: %v\n", r.City, r.OneWord, r.Celcius))
```

Comment:
`fmt.Sprintf` returns a string, so this explicit `string()` conversion must be unnecessary, mustn't it?

Reply:
fixed